content/new-content-banner.md

```diff
@@ -1,6 +1,6 @@
 ---
-show: false
-name: "LBJ Report "
-text: 'TJI data featured in "COVID and Corrections" report '
+show: true
+name: Explore the Data page
+text: New "Explore the Data" page showcases TJI data tools
```

Comment:
We don't actually say "explore the data" anywhere on that page, currently. I wonder if we should change the name of the link in the header?

The banner already includes a "new" indicator at the beginning, so we could dump that.

```suggestion
text: Browse all of TJI‘s interactive data tools
```